papaya-web/src/main/resources/application-prod.properties

```diff
@@ -14,3 +14,14 @@ spring.mail.port=${PAPAYA_EMAIL_PORT}
 spring.mail.properties.mail.smtp.starttls.enable=true
 notifications.cron.trigger=0 * * * * ?
 notifications.frequency.minutes=1440
+keycloak.realm=papaya
```

Comment:
Te linijki są takie same w obu propertiesach? Jak się da w zwykłym `application.properties` bez profilu to nie wystarczy?

Reply:
tej konfiguracji nie może być w testach a na prodzie i na devie mogą być potencjalnie różne, na razie są takie same dla zmyłki